non-pinned.json

```diff
@@ -13,6 +13,7 @@
   "prConcurrentLimit": 0,
   "prHourlyLimit": 0,
   "branchConcurrentLimit": 0,
+  "rebaseWhen": "conflicted",
```

Comment:
What's wrong with the default auto?

> auto: Renovate will autodetect the best setting. It will use behind-base-branch if configured to automerge or repository has been set to require PRs to be up to date. Otherwise, conflicted will be used instead

https://docs.renovatebot.com/configuration-options/

Reply:
The negative is that you expect a different behaviour when those settings are enabled in the repository. I'd probably prefer we set this in specific repos explicitly for when auto doesn't work.

Reply:
The more important question to me is "why rebase when _not_ conflicted?". Independent of this not working as documented, why do we want to rebase outside of conflicts? It just seems unnecessary, and I don't want to be held to whatever Renovate does when we don't want it. This change can come back, here and org-wide.